### ISSUE TITLE
Manual catalog scanning

### ISSUE THREAD

**Issue by TRtomasz** [Stremio/stremio-local-addon#29]
**Description**
Users are  missing feature to scan folders manually. Indexing on windows is limited and users who has NAS are not able to watch movies that are stored on these devices. We should add option to add folders that should be scanned that could be added as stremio setting but it is not necessary as i consider it a Power User setting, so they should be able to do it by themselves

**Comment by dufferzafar** [comment]
How is this addon even working if this setting is not available?

**Comment by Ivshti** [comment]
it uses the search index (only on Mac/Windows) to quickly list all video files on the system. On Linux, it just runs a `find` on preset directories such as ~/Downloads and ~/Videos

**Comment by dufferzafar** [comment]
um, it doesn't seem to do it on ~/Videos i'm sure, because i have some videos there and it doesn't pick them up!

**Comment by Rikj000** [comment]
Fully agree that this necessary.

If looking through all open issues,   
this is by far the most requested/discussed topic for this add-on so far.

### Issue Description
A lot of users, me included, struggle with being able to detect local files.   
The addon returns "Oops! This addon returned empty content."

### Enhancement Description
Give users the ability to configure multiple local paths to scan for local content.

This should:
- Help resolve issues with users being unable to locate local files
- Improve scanning performance in general, due to only scanning the required folders

### Related issues
- https://github.com/Stremio/stremio-local-addon/issues/37
- https://github.com/Stremio/stremio-local-addon/issues/34
- https://github.com/Stremio/stremio-local-addon/issues/33
- https://github.com/Stremio/stremio-local-addon/issues/30

### Environment
- **OS:** Manjaro Linux x86_64 KDE
- **Kernel:** 5.15.78-1-MANJARO
- **StremIO:** 4.4.159-1 - AUR
- **Local Files add-on:** 1.10.0

**Comment by SplitMilky** [comment]
Yeah I would love a section to add the folders I want included.  I have 7 hard drives on this machine and well none of my movies or shows show up.

**Comment by hugo9655** [comment]
I wouldnt consider this to be a power user setting, just basic functionality imo. I store my media on a separate drive and always have to drag and drop. I'd love to see this implemented so stremio can always watch my movies folder.

**Comment by Nimrod-P** [comment]
Not to put pressure on anyone... 
Does anyone know about a fix/update that can sort this?...
